model_evaluation/jigsaw_evaluation_pipeline.ipynb

```diff
@@ -339,7 +226,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install -U -q git+https://github.com/conversationai/unintended-ml-bias-analysis"
+    "# User inputs: Model description\n",
+    "TEXT_FEATURE_NAME = 'comment_text' #Input text\n",
+    "SENTENCE_KEY = 'comment_key' #Input key\n",
```

Comment:
Does it make sense to add a slightly more in-depth comment explaining what SENTENCE_KEY is and whether the user needs to change the default?

Reply:
Good point, I added some comments and will resolve this issue with new PR. The first two arguments will be visible from run.py

Reply:
Updated comments.